oracle-plugin/src/e2e-test/features/sink/OracleDesignTime.feature

```diff
@@ -0,0 +1,93 @@
+#
+# Copyright © 2023 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+@Oracle
+Feature: Oracle sink - Verify Oracle sink plugin design time scenarios
+  @Oracle_Required
+  Scenario: To verify Oracle sink plugin validation errors for mandatory fields
```

Comment:
Validation scenarios should be part of the Validation feature file

Reply:
Removed from OracleDesignTime.feature and added in OracleDesignTimeValidation.feature file.